components/Common/Preview/index.tsx

```diff
@@ -1,46 +1,34 @@
 import classNames from 'classnames';
 import Image from 'next/image';
-import type { CSSProperties, ComponentProps, FC, ReactNode } from 'react';
-
-import { useRouter } from '@/hooks/useRouter';
+import type { FC, ReactNode } from 'react';
 
 import styles from './index.module.css';
 
 type PreviewProps = {
   type?: 'announcement' | 'release' | 'vulnerability';
   title: ReactNode;
-  height?: CSSProperties['height'];
-  width?: CSSProperties['width'];
-} & Omit<ComponentProps<'div'>, 'children'>;
+  className?: string;
+};
 
 const Preview: FC<PreviewProps> = ({
   type = 'announcement',
   title,
-  height = 630,
-  width = 1200,
-  ...props
```

Comment:
Is it better to maintain the forwarding of props to the `div` element instead of specifying `className` prop directly? I think it is more flexible that way in case we need to pass in additional props to the root element. :)

Reply:
When I examined the pages in Figma, I saw that the only change in usage in different places would be the styling, so I arranged them this way.

I think that preparing components for a specific purpose and as simply as possible will be very helpful in terms of maintenance/usage. We can add new props when we need them in the future 🤓